samples/Nancy.Swagger.Demo/Modules/ServiceDetailsMetadataModule.cs

```diff
@@ -71,7 +71,7 @@ public ServiceDetailsMetadataModule(ISwaggerModelCatalog modelCatalog, ISwaggerT
 
             RouteDescriber.DescribeRouteWithParams<SwaggerFile>("PostCustomerReview", "", "Add a customer's review", new[]
             {
-                new HttpResponseMetadata<SwaggerFile> { Code = 20, Message = "Review Added"},
+                new HttpResponseMetadata<SwaggerFile> { Code = 200, Message = "Review Added"},
```

Comment:
Whoops, I totally saw this and was like "oh I didn't realize there's a status code 20" and didn't think to verify